rules-reviewed/quarkus/springboot/tests/springboot-di-to-quarkus.windup.test.xml

```diff
@@ -18,6 +18,18 @@
                     <fail message="[springboot-di-to-quarkus-extension] SpringBoot Dependency artifact replaced by Quarkus extension was not found!" />
                 </perform>
             </rule>
+            <rule id="springboot-di-to-quarkus-00003-test">
+                <when>
+                    <not>
+                        <iterable-filter size="9">
```

Comment:
Why are there 9 'hits'? There are only 5 classes implementing the classes we're searching for.

Reply:
actually it's ok, I've worked it out. Some of the classes we search for have some of the others as sub-interfaces so when you implement one like that you get more than one hit